### PR TITLE
Pull unasked coda questions, closes #199

### LINKED ISSUE #199: Pull unasked questions from coda to ask on Discord
We'd like the bot to pull unasked questions from coda to prompt discussions.

Unstarted from this table: https://coda.io/d/AI-Safety-Info_dfau7sl2hmG/Write-answers_suuwH#Write-answers_tu4_x/r54

Pull the question with the oldest AskedOnDiscord field (random if multiple), post the title of Edit Answer to #general along with the Gdocs link which is the link of Edit Answer, then set that AskedOnDiscord to `Today()`. I can make a new view on the data which provides Name and Link easily enough if that's helpful.

### PR DESCRIPTION
**Pull request by MatthewBaggins**

### CODE REVIEW

**Review by Aprillion — APPROVED** [review]
Why do we need 2 notebooks and only 1 of them being in `notebooks` folder?

File: modules/questions.py [hunk 1]
```diff
@@ -1,129 +1,450 @@
...
+load_dotenv()
 
-class QuestionQueueManager(Module):
-    """Module to manage commands about the question queue"""
 
-    EMPTY_QUEUE_MESSAGE = "There are no questions in the queue"
-    
-    def __init__(self):
-        super().__init__()
+class QuestionManager(Module):
```
[the remark this answers is not in the document]
Comment (MatthewBaggins) [reply modules/questions.py:33]: > Why do we need 2 notebooks and only 1 of them being in `notebooks` folder?

We don't need any. Thanks, I'll delete them for next commit

File: modules/questions.py [hunk 2]
```diff
@@ -1,129 +1,450 @@
...
+    (
+    (\#|n|num)\s+(of\s+)?({question_query})
+    )
+|
+    (\#\s*q)|(nq) # shorthands, you can just ask "nq" for number of questions
+)
+\?* # optional question mark
+$   # end
+""".format(
+    question_query=_PAT_QUESTION_QUERY
```
[the remark this answers is not in the document]
Comment (robertskmiles) [reply modules/questions.py:463]: I love exrex for this, you can give it a regex and it generates things that match it

**Comment by robertskmiles** [comment]
Also are we sure that the coda api token is being handled correctly so this will work in production? Presumably we have to add a github secret or something?

File: modules/questions.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Aprillion) [reply modules/questions.py:463]: yeah, but when I don't know what to tell stampy to make him say how many points I have or to give me next question, I won't have a regex at hand for which I don't know what will match it

when I will be just frustrated and try to look around source code, I can guarantee you that I won't be putting all of the regexes to some external tool 😅

**Comment by Aprillion** [comment]
> Also are we sure that the coda api token is being handled correctly so this will work in production? Presumably we have to add a github secret or something?

Matthew asked me to add new GH secret for coda token, but there seems to be more wiring needed to make it visible as env variable .. might be easiest for @robertskmiles  to add it as env variable on the server and we should discuss how to handle stampy secrets during office hours.

**Pull request merged**